.github/workflows/formatter/formatter_code.jl

```diff
@@ -11,7 +17,8 @@ for main_path in main_paths
             remove_extra_newlines = true,
             verbose = true,
             always_for_in = true,
-            whitespace_typedefs = true
+            whitespace_typedefs = true,
+            always_use_return = true
```

Comment:
Just to confirm: this needs to be set to false, right?

Reply:
correct